.github/workflows/pre-merge_actions.yml

```diff
@@ -38,7 +38,6 @@ jobs:
         run: |
           git config --global user.name "${GITHUB_ACTOR}";
           git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
why is this taken out? what did it do in the first place?

Reply:
That was introduced this morning by me when I was trying to solve the same problem. It's basically what git was suggesting to do in the error for the bumpversion, but adding that line did not work.